src/descriptor/segwitv0.rs

```diff
@@ -374,15 +374,23 @@ impl<Pk: MiniscriptKey> Wpkh<Pk> {
 impl<Pk: MiniscriptKey + ToPublicKey> Wpkh<Pk> {
     /// Obtains the corresponding script pubkey for this descriptor.
     pub fn script_pubkey(&self) -> ScriptBuf {
-        let addr = Address::p2wpkh(&self.pk.to_public_key(), Network::Bitcoin)
+        use core::convert::TryFrom;
```

Comment:
Can't we move this up to the imports? It also repeated down a little bit in L388.

Reply:
Yes, definitely. I must have thrown this up just roughly hacked together, I tend to put use statements in functions when I'm doing quick and dirty work. I see from the PR description that I did not indicate that was the case at all, so apologies for stealing your review time.